package.json

```diff
@@ -31,7 +31,8 @@
     "versions": "./bin/plugin/cli.js versions",
     "build": "wp-scripts build",
     "build-plugins": "npm-run-all 'build:plugin:!(performance-lab)'",
-    "build:plugin:performance-lab": "rm -rf build/performance-lab && mkdir -p build/performance-lab && git archive HEAD | tar -x -C build/performance-lab",
+    "build-plugins:zip": "npm-run-all 'build:plugin:!(performance-lab) --env zip=true' 'build:plugin:performance-lab'",
+    "build:plugin:performance-lab": "rm -rf build/performance-lab && mkdir -p build/performance-lab && git archive HEAD | tar -x -C build/performance-lab && cd build && zip -qr performance-lab.zip performance-lab",
```

Comment:
TODO: Remove it once https://github.com/WordPress/performance/pull/1182 is merged.